montydb/database.py

```diff
@@ -5,8 +5,8 @@
 
 
 class MontyDatabase(BaseObject):
+
     def __init__(self, client, name, codec_options=None, write_concern=None):
-        """ """
```

Comment:
An empty docstring serves no purpose.

If your ide is complaining it is missing, put in a value but not an empty one.